pkg/network/events/network_consumer.go

```diff
@@ -0,0 +1,45 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+// +build linux
+
+package events
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/eventmonitor"
+	smodel "github.com/DataDog/datadog-agent/pkg/security/secl/model"
+)
+
+// NetworkConsumer describes a process monitoring object
+type NetworkConsumer struct{}
+
+func (n *NetworkConsumer) Start() error {
+	return nil
+}
+
+func (n *NetworkConsumer) Stop() {
+}
+
+// ID returns id for process monitor
+func (n *NetworkConsumer) ID() string {
+	return "NETWORK"
+}
+
+// NewNetworkConsumer returns a new NetworkConsumer instance
+func NewNetworkConsumer(evm *eventmonitor.EventMonitor) (*NetworkConsumer, error) {
+	h := Handler()
```

Comment:
Looks like this has not been hooked up to the actual handler. Do we have to do this as a follow on? https://github.com/DataDog/datadog-agent/blob/main/pkg/network/events/monitor.go#L43

Reply:
Oh nvm, it is hooked up.